libs/sdk-backend-mockingbird/src/recordedBackend/dashboards.ts

```diff
@@ -54,9 +94,9 @@ export class RecordedDashboards implements IWorkspaceDashboardsService {
         });
 
         return Promise.resolve(result);
```

Comment:
This should probably return `localDashboards` too?

Reply:
i was thinking about it but then decided against. the reasoning was to reduce impact that these different 'ad-hoc' dashboards may have on other tests. it can mess up assertions depending on the order of the tests etc.

having the saved dashboard available only when test knows about it is safer

Reply:
Makes sense, thanks.